src/integrationTest/kotlin/com/sourcegraph/cody/util/CodyIntegrationTextFixture.kt

```diff
@@ -259,7 +266,7 @@ open class CodyIntegrationTextFixture : BasePlatformTestCase() {
   }
 
   companion object {
-    const val ASYNC_WAIT_TIMEOUT_SECONDS = 10L
+    const val ASYNC_WAIT_TIMEOUT_SECONDS = 30L
```

Comment:
Why is that needed? In the replay mode whole test should finish in well under 10s, not to mention single call.
Or maybe is it needed for the recording mode?
If so, maybe we should have configurable timeouts in the gradle config?

Reply:
Reverted